pep-0632.rst

```diff
@@ -4,6 +4,7 @@ Author: Steve Dower <steve.dower@python.org>
 Discussions-To: https://discuss.python.org/t/pep-632-deprecate-distutils-module/5134
 Status: Accepted
 Type: Standards Track
+Topic: Packaging
```

Comment:
This PEP is standard library, not packaging. It doesn't say anything about what packaging projects should do, and restricts itself to the changes in the stdlib.